src/components/Routes.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import Login from "../pages/Login";
+import Home from "../pages/Home";
+import Work from "../pages/Work";
+import About from "../pages/About";
+import Contact from "../pages/Contact";
+import NotFound from "../pages/NotFound";
+
+class Routes extends React.Component<any, any> {
+    render() {
+        return (
+            <div>
+                <Switch>
+                    <Route exact={true} path="/" component={Home}/>
```

Comment:
This is by no means a full review, but I just wanted to point out that we can't use absolute paths like these because they break page refresh. For technical reasons, the latest release candidate builds and pull request builds are stored in separate directories on GitHub pages. So clicking "about" on https://dubious-developments.github.io/pull-request-10/index.html sends the user to https://dubious-developments.github.io/about . Page refresh then takes the user straight to a 404, which is no bueno.

Can't we just use relative paths everywhere?

Reply:
I will discuss this tomorrow with the frontend team

Reply:
All right!